packages/plugins/model-evaluate/image-generation-tensorflow-model-evaluate/src/index.ts

```diff
@@ -1,7 +1,7 @@
-import { ModelEvaluateType, UniModel, ImageDataset, EvaluateResult, ArgsType } from '@pipcook/pipcook-core';
+import { ModelEvaluateType, UniModel, ImageDataset, EvaluateResult } from '@pipcook/pipcook-core';
 
 const cycleGanModelEvaluate: ModelEvaluateType
-  = async (data: ImageDataset, model: UniModel, args: ArgsType): Promise<EvaluateResult> => {
+  = async (data: ImageDataset, model: UniModel): Promise<EvaluateResult> => {
```

Comment:
This is not compatible with the type [`ModelEvaluateType`](https://github.com/alibaba/pipcook/blob/master/packages/core/src/types/plugins.ts#L176-L178) yet, we need some updates on the interface.

Reply:
@FeelyChau May I ask you to fix it in another pull request?

Reply:
OK, I'll change the `args` to optional.

Reply:
I checked the document of ts function here:
https://www.typescriptlang.org/docs/handbook/type-compatibility.html#comparing-two-functions
as the doc said, it's valid to pass `(data: ImageDataset, model: UniModel): Promise<EvaluateResult>` to `(data: ImageDataset, model: UniModel, args: ArgsType): Promise<EvaluateResult>`.

Reply:
Do we have a way to make this fail?

Reply:
This is valid in ts, and very useful, like the example in the document:
```js
let items = [1, 2, 3];

// Don't force these extra parameters
items.forEach((item, index, array) => console.log(item));

// Should be OK!
items.forEach(item => console.log(item));
```
If we make this fail, the case above will be failed too.